build.gradle

```diff
@@ -657,6 +657,47 @@ project(':iceberg-delta-lake') {
   }
 }
 
+project(':iceberg-bigquery') {
+  test {
+    useJUnitPlatform()
+  }
+
+  dependencies {
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    implementation project(':iceberg-gcp')
+
+    implementation platform(libs.google.libraries.bom)
+    compileOnly "com.google.cloud:google-cloud-storage"
+    implementation "com.google.cloud:google-cloud-bigquery"
+    implementation "com.google.cloud:google-cloud-core"
+
+    api project(':iceberg-api')
+    implementation project(':iceberg-common')
+    implementation project(':iceberg-core')
+
+    compileOnly libs.hadoop3.common
+    compileOnly("org.apache.hive:hive-metastore:4.0.1") {
```

Comment:
this should be defined in `libs.versions.toml` as `hive4-metastore` and a separate `hive4` that points to that version